testing/fstest/fstest_main.c

```diff
@@ -110,8 +111,10 @@ static void fstest_showmemusage(struct mallinfo *mmbefore,
                                    mmafter->arena    - mmbefore->arena);
   printf("ordblks  %8d %8d %8d\n", mmbefore->ordblks , mmafter->ordblks,
                                    mmafter->ordblks  - mmbefore->ordblks);
+#ifdef __NuttX__
```

Comment:
Why this is needed?

Reply:
Because mxordbk is a specific implementation in struct mallinfo

Reply:
we want building teston Linux.